src/Phinx/Seed/AbstractSeed.php

```diff
@@ -162,11 +162,21 @@ public function fetchAll($sql)
      */
     public function insert($table, $data)
     {
-        // convert to table object
-        if (is_string($table)) {
-            $table = new Table($table, [], $this->getAdapter());
-        }
-        $table->insert($data)->save();
+        $tableInstance = new Table($table, [], $this->getAdapter());
+        $tableInstance->insert($data)->save();
+    }
+
+    /**
+     * @param string $tableName Table name.
+     *
+     * @return bool
+     */
+    public function hasData(string $tableName): bool
+    {
+        $countQuery = $this->getAdapter()->query('SELECT COUNT(*) FROM ' . $tableName);
```

Comment:
This will need to call down to the adapter to quote the table name (and potential schema prefix).